charts/gridproxy/README.md

```diff
@@ -30,17 +24,10 @@
 
 - Apply certificate `kubectl create -f prod_issuer.yaml`
 
-- If you want to add more peers add using `--set env.PEERS="  tls:\\\/\\\/62.210.85.80:39575\\\n   tls:\\\/\\\/54.37.137.221:11129\\\n"` add `\\\` as escape characters before each `/`
-
 - Install the chart
 
-  **Note**: EXPLORER_URL, SERVER_IP and REDIS_URL has default values you may not pass them if you want to use the defaults
+  **Note**: EXPLORER_URL, SERVER_IP has default values you may not pass them if you want to use the defaults
```

Comment:
What is EXPLORER_URL?

Reply:
not used anywhere on the server or the image. should be removed, also SERVER_IP is not used, should be  SERVER_PORT. I think the docs need a full revision.